Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h

```diff
@@ -5,6 +5,18 @@
 
 #include "SpatialGDKEditorToolbarSettings.generated.h"
 
+USTRUCT()
+struct FInteropClassList
```

Comment:
Maybe something like `InteropTypebindingInfo` since each one of these will become a typebinding?
I just don't like `ClassList` since once instance of this only has one class and a list of includes.